x-pack/test_serverless/api_integration/test_suites/common/console/proxy_route.ts

```diff
@@ -25,14 +25,12 @@ export default function ({ getService }: FtrProviderContext) {
     describe('system indices behavior', () => {
       it('does not forward x-elastic-product-origin', async () => {
         // If we pass the header and we still get the warning back, we assume that the header was not forwarded.
-        return await supertestWithoutAuth
+        const response = await supertestWithoutAuth
           .post('/api/console/proxy?method=GET&path=/.kibana/_settings')
           .set('kbn-xsrf', 'true')
           .set(svlCommonApi.getInternalRequestHeader())
-          .set(roleAuthc.apiKeyHeader)
-          .then((response) => {
```

Comment:
Alternatively we can import `Response` type from `supertest`, but I believe we shouldn't import lib directly in tests. Open for discussion.